demo/src/configs/arsaboo/lovelace.ts

```diff
@@ -465,4 +465,5 @@ export const demoLovelaceArsaboo: DemoConfig["lovelace"] = (localize) => ({
       ],
     },
   ],
+  append_view_title: false,
```

Comment:
We should not do this on every config. You should check if it is true. If its False or undefined then it should not append